synapse/events/snapshot.py

```diff
@@ -57,6 +57,9 @@ class EventContext:
             If ``state_group`` is None (ie, the event is an outlier),
             ``state_group_before_event`` will always also be ``None``.
 
+        delta_before_after: If `state_group` and `state_group_before_event` are not None
+            then this is the delta of the state between the two groups.
```

Comment:
Is there a better name we can use? Maybe `state_delta_at_event` or something?

Reply:
It's hard because I want to make sure people don't think that its the state delta since the `prev_group`

Reply:
Right! Just the `before_after` I find confusing. Before/after *what*?!

Reply:
Oh i see. Yes. Ermh, `delta_before_and_after_event`?